cmd/containerd/main.go

```diff
@@ -164,7 +165,7 @@ high performance container runtime
 			case syscall.SIGUSR1:
 				dumpStacks()
 			default:
-				logrus.WithField("signal", s).Info("containerd: stopping GRPC server")
+				logrus.WithField("signal", s).Infof("containerd: stopping GRPC server after receiving %s", s)
```

Comment:
you don't need the signal in the `Infof` because its already added with `WithField`